index.js

```diff
@@ -132,6 +132,21 @@ function createQuestions(config) {
           : 'Provide a longer description:',
       when: !config.skipQuestions.includes('body')
     },
+    {
+      type: 'confirm',
+      name: 'isBreaking',
+      message: 'Are there any breaking changes?',
+      default: false
+    },
+    {
+      type: 'input',
+      name: 'breakingBody',
+      message:
+        'A BREAKING CHANGE commit requires a body. Please enter a longer description of the commit itself:\n',
+      when: answers => answers.isBreaking,
```

Comment:
I would like to make this step optional as ultimately `cz-emoji` is not bound to Conventional commits.

```suggestion
      when: !config.skipQuestions.includes('breaking'),
```

Reply:
I'm not sure what `breaking` in

```js
when: !config.skipQuestions.includes('breaking'),
```

referring to.

Reply:
It's referring to the name of the question to be skipped.
In the current implementation it's `breakingBody` actually, but I believe `breaking` would be just fine, especially if we remove the `isBreaking` question.